code/modules/vore/appearance/sprite_accessories_vr.dm

```diff
@@ -112,7 +112,7 @@
 	desc = ""
 	icon_state = "fenears"
 
-/datum/sprite_accessory/ears/sergal
+/datum/sprite_accessory/ears/sergal //Redundant
```

Comment:
Redundant how?

Reply:
Essentially a white uncolorable dupe for the colorable version, which in the multiply format will look just same in white tint.